packages/flutter/lib/src/cupertino/form_row.dart

```diff
@@ -0,0 +1,203 @@
+///             CupertinoFormRow(
+///               prefix: Text('Toggle'),
+///               helper: Text('Use your instincts'),
+///               error: toggleValue ? Text('Can't be true') : null,
+///               child: CupertinoSwitch(
+///                 value: this.toggleValue,
+///                 onChanged: (value) {
+///                   setState(() {
+///                     this.toggleValue = value;
+///                   });
+///                 },
+///               ),
+///             ),
+///           ],
+///         ),
+///       ),
+///     );
+///   }
+/// }
+/// ```
+/// {@end-tool}
+class CupertinoFormRow extends StatelessWidget {
+  /// Creates an iOS-style split form row with a standard prefix and child widget.
+  /// Also provides a space for error and helper widgets that appear underneath.
+  ///
+  /// The [child] parameter is required. This widget is displayed at the end of
+  /// the row.
+  ///
+  /// The [prefix] parameter is optional and is displayed at the start of the
+  /// row. Standard iOS guidelines encourage passing a [Text] widget to [prefix]
+  /// to detail the nature of the row's [child] widget.
+  ///
+  /// The [margins] parameter is used to pad the contents of the row. It defaults
+  /// to the standard iOS padding. If no edge insets are intended, explicitly
+  /// pass [EdgeInsets.zero] to [margins].
+  ///
+  /// The [helper] and [error] parameters are both widgets targeted at displaying
+  /// more information about the row. Both widgets are placed underneath the
+  /// [prefix] and [child]. When a [Text] is given to [error], it will be shown in
+  /// [CupertinoColors.destructiveRed] coloring and medium-weighted font.
+  const CupertinoFormRow({
+    Key? key,
+    required this.child,
+    this.prefix,
+    this.margins,
+    this.helper,
+    this.error,
+  }) : super(key: key);
+
+  /// A widget that is displayed at the start of the row.
+  ///
+  /// The [prefix] parameter is displayed at the start of the row. Standard iOS
+  /// guidelines encourage passing a [Text] widget to [prefix] to detail the
+  /// nature of the row's [child] widget.
+  final Widget? prefix;
+
+  /// Margins used to pad the contents of the row.
+  ///
+  /// Defaults to the standard iOS padding for form rows. If no edge insets are
+  /// intended, explicitly pass [EdgeInsets.zero] to [margins].
+  final EdgeInsetsGeometry? margins;
+
+  /// A widget that is displayed underneath the [prefix] and [child] widgets.
+  ///
+  /// The [helper] appears in primary label coloring, and is meant to inform the
+  /// user about interaction with the child widget.
+  final Widget? helper;
+
+  /// A widget that is displayed underneath the [prefix] and [child] widgets.
+  ///
+  /// The [error] widget is primarily used to inform users of input errors. When
+  /// a [Text] is given to [error], it will be shown in
+  /// [CupertinoColors.destructiveRed] coloring and medium-weighted font.
+  final Widget? error;
```

Comment:
Not a reason for it tbh. I just thought it might be useful to include both. (Say if you want to have the helper on at all times but then the error appears when validated.) Let me know if you think I should remove it. I showed it to @xster a couple of days ago